packages/kitchen/src/pan/Alerts.vue

```diff
@@ -90,6 +90,41 @@
           This is an error alert.
         </v-alert>
       </core-section>
+      <!-- Prominent & dense -->
+      <core-sectionx>
```

Comment:
should be <core-section>

Reply:
@aburai thanks, will fix

Reply:
@aburai there isn't this line in alerts pan